xarray/test/test_plot.py

```diff
@@ -2,6 +2,16 @@
 from __future__ import division
 from __future__ import print_function
 
+try:
+    import matplotlib as mpl
+    # Using a different backend makes Travis CI work
+    # This has to happen before other mpl imports
+    mpl.use('Agg')
+    # Order of imports is important here.
+    import matplotlib.pyplot as plt
+except ImportError:
+    pass
```

Comment:
@fmaussion Maybe just update the comments to clarify what the order should be. For example:

```
# Import matplotlib before xarray plotting
try:
     import matplotlib as mpl
     # Using a different backend makes Travis CI work
     mpl.use('Agg')
     import matplotlib.pyplot as plt
 except ImportError:
     pass
```